examples/vpc-scenario-2-nat-instances-per-az/main.tf

```diff
@@ -81,10 +81,13 @@ module "nat-instances" {
   source      = "../../modules/ec2-nat-instance"
   name_prefix = "${var.name}"
   key_name    = "${aws_key_pair.main.key_name}"
+
   # let AWS set IPs for us
   private_ips = []
+
   # list of subnets to deploy NAT instances into
-  public_subnet_ids    = ["${module.public-subnets.ids}"]
+  public_subnet_ids = ["${module.public-subnets.ids}"]
+
```

Comment:
I find the previous version "more aligned" and preferable to lots of spaces simply b/c we have a comment to doc the param. `fmt's` default here is to leave each line's `=` to fall where it falls, and to add extra whitespace between the lines. This is counter-intuitive to me.